app/components/form-security/template.hbs

```diff
@@ -156,12 +228,44 @@
   </div>
 
   <div class="col-sm-12 col-md-8">
-    {{form-key-value
-      initialMap=instance.logConfig.config
-      changed=(action (mut instance.logConfig.config))
-      addActionLabel="formSecurity.logConfig.addActionLabel"
-      keyPlaceholder="formSecurity.logConfig.keyPlaceholder"
-      valuePlaceholder="formSecurity.logConfig.valuePlaceholder"
-    }}
+    {{#if editing}}
+      {{form-key-value
+        initialMap=instance.logConfig.config
+        changed=(action (mut instance.logConfig.config))
+        addActionLabel="formSecurity.logConfig.addActionLabel"
+        keyPlaceholder="formSecurity.logConfig.keyPlaceholder"
+        valuePlaceholder="formSecurity.logConfig.valuePlaceholder"
+      }}
+    {{else}}
+      {{#if hasLogConfig}}
```

Comment:
This (and other places) seems like showing a table should be the `editable=false` behavior of `form-key-value`, instead of replicating in each component?